dist/injector.js

```diff
@@ -120,13 +120,13 @@ window.FAI.bundleHost = window.FAI.bundleHost || '/';
             i = 0,
             j = 0,
             chunkNames,
-            features;
-        for (i = 0; i < chunks.length; i++) {
-            features = chunks[i];
-            chunkNames = features.chunkNames;
+            asset;
+        for (i = 0; i < assets.length; i++) {
+            asset = assets[i];
+            chunkNames = asset.chunkNames;
             for (j = 0; j < chunkNames.length; j++) {
                 if (chunkNames[j] === feature) {
-                    url = NS.bundleHost + features.name;
+                    url = NS.bundleHost + asset.name;
                     if (video) {
                         url = url + '?version=latest&client=expansion';
                     }
```

Comment:
This video conditional will need to be handled in a separate PR. Not sure how we can handle it just yet.